### ISSUE TITLE
face_recognition_sface_2021dec output is always 

### ISSUE THREAD

**Issue by lynnwilliam** [opencv/opencv_zoo#114]
**Bug**

output matrix is always
rows=1 cols=128
[-0.13746102154254913] null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null ]

**Configuration** 

face_recognition_sface_2021dec.onnx
OpenCV arch x86_64
OpenCv V 4.3.0-1
Interface OpenCV Java
Mac v 12.6
OS MAC Monterey ( but same happens when I run this on windows machines and other Mac books).

Important: all my source faces are ALREADY ALIGNED and Cropped, using faceRecognizer.cropAlign(...) is NOT want I want to do.
and I don't know if that step is important, 
it's NOT documented that the step faceRecognizer.cropAlign(...) is important, so it can be skipped in my case.
But all my images are already copped and aligned.

**Test Image **

I am passing the image at 
https://facemri.com/Sarah0_aligned.jpg
( note its already aligned and face extracted).
But it happens with any image

**-- start java code**
FaceRecognizerSF faceRecognizer = FaceRecognizerSF.create("face_recognition_sface_2021dec.onnx", "");

Mat loadedImage = Imgcodecs.imread("Sarah0_aligned.jpg");

Mat colorMat = new Mat();
 Imgproc.cvtColor(loadedImage, colorMat, Imgproc.COLOR_BGRA2BGR);
  Mat resizeMat = new Mat();
  Imgproc.resize(colorMat, resizeMat, new Size(128,128));  //

Mat featureCanidate = new Mat();
log("resizeMat= " + resizeMat.size());
faceRecognizer.feature(resizeMat, featureCanidate);

**The current output** 
and the output matrix is always
rows=1 cols=128
[-0.13746102154254913] null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null null ]

**The Expected Output** 

each value in the 128d array should be fulled with numbers.

**Comment by fengyuentau** [comment]
`FaceRecognizerSF` is available since opencv>=4.5.4. So you should upgrade your opencv. By the way, I tried your image with the following python code:

```python
import numpy as np
import cv2 as cv

net = cv.FaceRecognizerSF.create("face_recognition_sface_2021dec.onnx", "")

face = cv.imread("Sarah0_aligned.jpg", 1)
print(face)

out = net.feature(face)
print(out)
```

And the `out` is not null.

**Comment by fengyuentau** [comment]
Closing due to inactivity. Feel free to re-open this issue in case I missed something.
